### DISCUSSION TITLE
How can I stop hourly charges for Cloud Software

### DISCUSSION THREAD

**Discussion by jayvs6341** [mozilla/hubs#5138] [Hubs Cloud Questions] [answered]
I deployed my own Cloud to AWS.

I am following following link to reduce costs.

https://hubs.mozilla.com/docs/hubs-cloud-aws-costs.html#how-do-costs-work-for-hubs-cloud

Can I know, Am I able to stop hourly cost for Software when it is not in use.

**Comment by matthewbcool** [comment, marked as answer]
Use the cloud formation template to toggle your stack offline when not in use. 
https://hubs.mozilla.com/docs/hubs-cloud-aws-updating-the-stack.html
You can change various settings of your hub's stack by performing a stack Update. You will not experience any downtime when making these changes. To Update your stack:

- Select the stack in the CloudFormation console
- Go to Stack Actions -> Update Stack
- Choose "Use current template"
- Review the parameter selections and choose 'Update'

**Switch your hub into Offline Mode to save costs (and redirect to a URL)**

**Reply by jayvs6341** [reply]
Will it save Software cost as well ?
